<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllMembers</class>
 <widget class="QWidget" name="AllMembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AllMembers{
	background-color:black;
}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>311</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>236</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>43</number>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="background">
     <color alpha="251">
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Phone Number</string>
    </property>
    <property name="font">
     <font>
      <family>Arial Narrow</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="email_del">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border:1px solid black;
	border-radius:20px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#del_btn {
	background-color:rgb(255, 157, 7);
	border:1px solid black;
	border-radius:25px;
	color:black;
}
#del_btn:hover{
		background-color:black;
		color:rgb(255, 157, 7);
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upd_name">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border:1px solid black;
	border-radius:20px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_cheker">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>320</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border:1px solid black;
	border-radius:20px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upd_phone">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>420</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border:1px solid black;
	border-radius:20px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="upd_email">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border:1px solid black;
	border-radius:20px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upd_btn">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upd_btn {
	background-color:rgb(255, 157, 7);
	border:1px solid black;
	border-radius:25px;
	color:black;
}
#upd_btn:hover{
		background-color:black;
		color:rgb(255, 157, 7);
}</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(255, 157, 7);
}</string>
   </property>
   <property name="text">
    <string>Email for Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(255, 157, 7);
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(255, 157, 7);
}</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:rgb(255, 157, 7);
}</string>
   </property>
   <property name="text">
    <string>Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>51</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Avenir</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-left:1px solid rgb(255, 157, 7);
	border-top:1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
